crates/ruff_server/src/message.rs

```diff
@@ -32,7 +32,10 @@ pub(crate) fn init_messenger(client_sender: ClientSender) {
         }
 
         let backtrace = std::backtrace::Backtrace::force_capture();
-        tracing::error!("{panic_info}\n{backtrace}");
+        #[allow(clippy::print_stderr)]
+        {
+            eprintln!("{panic_info}\n{backtrace}");
+        }
```

Comment:
Did you change this to `eprintln` because it might happen before tracing is set up? I think it would still be nice if the message also goes to tracing to ensure the error is captured in a logfile if `logFile` is set

Reply:
Good idea!